watson-developer-cloud/discovery-components
# test: search results cypress tests

#### What do these changes do/fix?
Adding Cypress tests for the functionality surrounding search results. Functionality covered in these tests:
- [x] Table results
- [x] Passage results

This PR builds off of work that started with #26 and #18.


#### How do you test/verify these changes?
Run Cypress tests with the Cypress UI:
```
yarn workspace discovery-search-app cypress
```
OR:
Run Cypress tests without the Cypress UI:
```
yarn workspace discovery-search-app test:e2e
```
Either way, you should be able to verify that the new tests work and that they make sense for the functionality.
#### Have you documented your changes (if necessary)?
No.
#### Are there any breaking changes included in this pull request?
Nope.
